packages/mirta-i18n/README.ru.md

```
@@ -0,0 +1,240 @@
+pnpm add @mirta/i18n
+```
+
+⚠️ Этот пакет — часть внутренней инфраструктуры фреймворка Mirta. Обычно не используется напрямую.
+
+## Использование
+
+### 1. Подготовьте файлы локалей
+
+Организуйте папку `locales` следующим образом:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)